web/src/main/java/org/cbioportal/web/util/DataBinHelper.java

```diff
@@ -308,4 +301,35 @@ public String stripOperator(String value) {
     public boolean isAgeAttribute(String attributeId) {
         return attributeId != null && attributeId.matches("(^AGE$)|(^AGE_.*)|(.*_AGE_.*)|(.*_AGE&)");
     }
+
```

Comment:
I didn't see a DataBinHelperTest in the files - this seems like a good method to unit test.

Reply:
true, but looks like this scenario is tested while testing `DataBinner` in `DataBinnerTest.java`